### PR TITLE
Merge feature/version to master

### PR DESCRIPTION
**Pull request by tswhison**
Enables -v --version support for all the apps.

### CODE REVIEW

File: libopaecxx/CMakeLists.txt [hunk 1]
```diff
@@ -45,7 +45,7 @@ set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
 find_package(Threads)
 
 set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DHAVE_CONFIG_H -pthread")
```
Comment (r-rojo) [inline libopaecxx/CMakeLists.txt]: Do we still need this?

Comment (tswhison) [reply libopaecxx/CMakeLists.txt]: I think so, for the cxx sample apps.

File: tools/base/fpgaconf/fpgaconf.c [hunk 2]
```diff
@@ -123,7 +126,7 @@ void help(void)
 	       "        fpgaconf [-hvn] [-B <bus>] [-D <device>] [-F <function>] [-S <socket-id>] <gbs>\n"
 	       "\n"
 	       "                -h,--help           Print this help\n"
-	       "                -v,--verbose        Increase verbosity\n"
+	       "                -V,--verbose        Increase verbosity\n"
 	       "                -n,--dry-run        Don't actually perform actions\n"
```
Comment (HananiaAmir) [inline tools/base/fpgaconf/fpgaconf.c]: Do you want to add a line for -v for version ?

Comment (tswhison) [reply tools/base/fpgaconf/fpgaconf.c]: Doh, good catch! https://github.com/OPAE/opae-sdk/pull/1361

**Pull request merged**